src/Microsoft.AspNet.Razor/Properties/AssemblyInfo.cs

```diff
@@ -5,5 +5,4 @@
 using System.Runtime.CompilerServices;
 
 [assembly: InternalsVisibleTo("Microsoft.AspNet.Razor.Test")]
-[assembly: InternalsVisibleTo("Microsoft.AspNet.Razor.Runtime.Test")]
```

Comment:
These IVT attributes are about unit testing. Where is the 1:1 requirement in the guidelines?

This seems like an unrelated change mixed into an actual fix.